0001-typedb/02-dev/08-api.md

```diff
@@ -0,0 +1,141 @@
+---
+pageTitle: API & Drivers
+keywords: api, typedb, typeql, concept
+longTailKeywords: TypeDB API, concept api, client api
+Summary: TypeDB API and Drivers description.
+---
+
+# API & Drivers
+
+TypeDB servers accept gRPC calls from [TypeDB Clients](../../02-clients/00-clients.md). This connection has "smart" 
+client architecture, meaning TypeDB Clients perform validation of queries, load balancing, and some other operations 
+transparent for users.
+
+To access all features of TypeDB and TypeDB Clients use the default interface provided by the clients:
+
+* GUI from [TypeDB Studio](../../02-clients/01-studio.md),
+* CLI from [TypeDB Console](../../02-clients/02-console.md),
+* [Client API](../../02-clients/08-api.md) from TypeDB Drivers: 
```

Comment:
Where do we use driver and client?

Reply:
Everywhere we speak of them. But mostly on the Clients overview page. It is still Work in progress, because it's the Clients section of documentation.

Reply:
TypeDB Client is anything that can connect to a TypeDB server with gRPC. TypeDB Driver is a TypeDB Client in a form of a library that provides an API (Client API) to talk to a TypeDB server. So a TypeDB Client with API (not GUI, or CLI as opposed to Studio and Console).

Reply:
Makes sense to me.
Internally, We always just say client to refer to the drivers but yours definition does feel more standard.

Reply:
client is still a valid term but it's the most generic one. When we want to talk about libraries we specify them all together as drivers. Or TypeDB Drivers to be exact. You can check the very first page of TypeDB Clients section on our staging area - it already has some content on that matter.